internal/cli/clients/client.go

```diff
@@ -1712,6 +1780,13 @@ func (c *EpinioClient) curlWithCustomErrorHandling(endpoint, method, requestBody
 	return bodyBytes, nil
 }
 
+func (c *EpinioClient) TargetOk() error {
+	if c.Config.Org == "" {
+		return errors.New("Internal Error: No targeted")
```

Comment:
```suggestion
		return errors.New("Internal Error: No namespace targeted")
```

Reply:
This is an internal error, because we have the default in the config normally, right?

Reply:
Yes.